### ISSUE TITLE
Impliment accessibility and sapification support

### ISSUE THREAD

**Issue by king-dahmanus** [sveneschlbeck/Multi-Language-RTVC#21]
Hi devs. I'm not  a programmer, but I suggest making the windows app accessible to screen readers and assistive technologies. The other thing I suggest is making the app able to export the models as a sapi5 voice so assistive techs can use them. The most crucial requirement is responsiveness and realtime output, which means that the voice shouldn't lag before the speech or in the middle of inference. Please impliment these suggestions. What does this app use for guis? Wxpython is the most accessible one in my opinion, as it uses native controles which are accessible to assistive technologies. Thanks and I hope you impliment them.

**Comment by king-dahmanus** [comment]
also does this project have a discord server where the community lives?

**Comment by sveneschlbeck** [comment]
@king-dahmanus Thank you very much for your input. It's cool to see your interest. I'll check and get back on it. At the moment, I am pretty busy at uni, so the project is on a brief pause but I'll get on it asap

**Comment by sveneschlbeck** [comment]
Yes, there is a discord server. @Dannypeja Do you have the link?

**Comment by king-dahmanus** [comment]
thank you! Hope that this app becomes accessible to the blind and also its
models would be available to use with assistive technologies!

**Comment by Dannypeja** [comment]
here you go! https://discord.gg/asym7axdgd

**Comment by king-dahmanus** [comment]
I've joined the discord
